components/more_direct_channels/more_direct_channels.tsx

```diff
@@ -68,6 +69,7 @@ type Props = {
     recentDMUsers?: Array<UserProfile & {last_post_at: number }>;
     statuses: RelationOneToOne<UserProfile, string>;
     totalCount?: number;
+    prepopulatedValues?: [];
```

Comment:
can you type this more strictly? Just an array seems to vague. I think you do know what beforehand what can go in there, right? (I think in this case it is `users`)